readme.md

```diff
@@ -163,6 +163,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [Comprehensive XDP offload---Handling the edge cases](https://www.netdevconf.org/2.2/session.html?viljoen-xdpoffload-talk) - An update on the topic above.
 - [hBPF - eBPF in hardware](https://github.com/rprinz08/hBPF) - An eBPF CPU written for FPGAs.
 - [OpenCSD eBPF SSD offloading](https://github.com/Dantali0n/qemu-csd) - Computational Storage simulation (QEMU) platform with FUSE LFS filesystem for Zoned Namespaces NVMe SSDs using uBPF for compute kernel offloading, all in userspace.
+- [Delilah: eBPF-offload on Computational Storage](https://www.researchgate.net/publication/370818368_Delilah_eBPF-offload_on_Computational_Storage) - The first public description of an actual computational storage device supporting eBPF-based code offload.
```

Comment:
I'm not sure whether “first” and “public” make a lot of sense to the reader in the description of the project. Although I do understand you are (rightfully) proud of it, I'd keep it terse and just tell what this is. Also what does “actual” mean in the description?

Reply:
With _actual_, I mean it is not _virtualised_ or _emulated_. This is the only device that supports eBPF offload to storage on a physical device.

How does this sound:
_Delilah is a Computational Storage Processor (CSP) built for eBPF offload to storage devices._

Reply:
Thanks for the clarification! I'd use “physical” in that case, I find it more explicit than “actual”.

Anyway, yes your revised version looks good!